core/domain/collection_services.py

```diff
@@ -312,6 +313,38 @@ def get_multiple_collections_by_id(collection_ids, strict=True):
     return result
 
 
+def get_collection_and_collection_rights_by_id(collection_id):
+    """Returns a tuple for collection domain object and collection rights
+        object.
+
+    Args:
+        collection_id: str. Id of the collection.
+
+    Returns:
+        tuple(collection object or None, collection rights object or None).
```

Comment:
tuple(Collection, CollectionRights). The collection and collection rights domain objects, respectively.

Reply:
done.